<?xml version="1.0"?>
<robot name="leftArm">
  <joint name="Baxter_leftArm_joint1">
    <limit lower="-3.14" upper="3.14"/>
  </joint>
  <joint name="Baxter_leftArm_joint2">
    <limit lower="-3.14" upper="3.14"/>
  </joint>
  <joint name="Baxter_leftArm_joint3">
    <limit lower="-3.14" upper="3.14"/>
  </joint>
  <joint name="Baxter_leftArm_joint4">
    <limit lower="-3.14" upper="3.14"/>
  </joint>
  <joint name="Baxter_leftArm_joint5">
    <limit lower="-3.14" upper="3.14"/>
  </joint>
  <joint name="Baxter_leftArm_joint6">
    <limit lower="-3.14" upper="3.14"/>
  </joint>
  <joint name="Baxter_leftArm_joint7">
    <limit lower="-3.14" upper="3.14"/>
  </joint>  
</robot>
